ci-operator/step-registry/capi-migration-check/capi-migration-check-ref.metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+	"path": "capi-migration-check/capi-migration-check-ref.yaml",
+	"owners": {
+		"approvers": [
+			"sunzhaohua2",
+			"miyadav",
+			"huali9",
+			"shellyyang1989"
+		],
+		"reviewers": [
+			"sunzhaohua2",
+			"miyadav",
+			"huali9",
+			"shellyyang1989"
+		]
+	}
+}
```

Comment:
minor nit  , extra empty space.

Reply:
Checked other steps, there is no extra empty space as well.